src/System.Private.CoreLib/shared/System/Threading/Tasks/Task.cs

```diff
@@ -230,8 +231,11 @@ internal static bool AddToActiveTasks(Task task)
 
         internal static void RemoveFromActiveTasks(Task task)
         {
+            if (s_currentActiveTasks == null)
+                return;
```

Comment:
Is this check actually needed?  Seems like RemoveFromActiveTasks should only ever be called when AddToActiveTasks was previously called; they're supposed to come in pairs.  Is that not happening somewhere?

Reply:
It's quite hard to audit that they always come in part but no problem to streamline the code.

Reply:
If it's not easy to validate, then it's fine to leave the check.